model_info/enwiktionary.reverted.md

```diff
@@ -15,14 +15,14 @@ Model Information:
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-6-amd64'
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=91825):
+	counts (n=91796):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       385  -->       10       375
-		False    91440  -->        3     91437
+		True       384  -->       10       374
```

Comment:
I wonder why this model is so hesitant to predict true?

Reply:
Very very low positive rate I think